packages/amplify-category-geo/src/service-utils/placeIndexUtils.ts

```diff
@@ -22,7 +22,7 @@ export const createPlaceIndexResource = async (context: $TSContext, parameters:
   }
 
   // update the pricing plan
-  await updateGeoPricingPlan(context, parameters.pricingPlan);
+  // await updateGeoPricingPlan(context, parameters.pricingPlan);
```

Comment:
is this line still needed? if so we should check why the tests pass without it

Reply:
It is not needed for now. We are waiting for Location Service to support updating the pricing plan for Geo resources. Hence this is not tested extensively yet.